### PR TITLE
UP-4884: add unit test portal security remoting

### PR DESCRIPTION
**Pull request by stalele**
##### Checklist

-   [X] the [individual contributor license agreement][] is signed
-   [X] commit message follows [commit guidelines][]
-   [X] tests are included
-   [ ] documentation is changed or added
-   [ ] [message properties][] have been updated with new phrases
-   [ ] view conforms with [WCAG 2.0 AA][]

##### Description of change

Added unit test for package - Added unit test for org.apereo.portal.security.remoting package
Jira ticket - https://issues.jasig.org/browse/UP-4884

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by ChristianMurphy** [comment]
@stalele I'm going to close and re-open this PR to re-trigger AppVeyor.
Currently Appveyor is incorrectly showing
```
AppVeyor was unable to build non-mergeable
```
http://help.appveyor.com/discussions/problems/4830-stuck-on-appveyor-was-unable-to-build-non-mergeable

File: uPortal-api/uPortal-api-rest/src/test/java/org/apereo/portal/rest/PagsRESTControllerTest.java [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
-import junit.framework.Assert;
 import org.apereo.portal.groups.pags.dao.IPersonAttributesGroupDefinition;
 import org.apereo.portal.groups.pags.dao.PagsService;
 import org.apereo.portal.groups.pags.dao.jpa.PersonAttributesGroupDefinitionImpl;
 import org.apereo.portal.security.IPerson;
 import org.apereo.portal.security.IPersonManager;
 import org.apereo.portal.security.provider.PersonImpl;
 import org.apereo.portal.services.GroupService;
+import org.junit.Assert;
 import org.junit.Before;
```
Comment (bjagg) [inline uPortal-api/uPortal-api-rest/src/test/java/org/apereo/portal/rest/PagsRESTControllerTest.java]: Nice improvement

File: uPortal-api/uPortal-api-rest/src/test/java/org/apereo/portal/rest/layout/LayoutJsonV43RenderingControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import org.mockito.MockitoAnnotations;
+
+public class LayoutJsonV43RenderingControllerTest {
+
+    @InjectMocks LayoutJsonV43RenderingController controller;
+    @Mock private HttpServletRequest req;
+
+    @Mock private HttpServletResponse res;
+
+    @Mock private IPortalRenderingPipeline portalRenderingPipeline;
```
Comment (bjagg) [inline uPortal-api/uPortal-api-rest/src/test/java/org/apereo/portal/rest/layout/LayoutJsonV43RenderingControllerTest.java]: Spacing?

File: uPortal-api/uPortal-api-rest/src/test/java/org/apereo/portal/rest/layout/LayoutJsonV1RenderingControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+public class LayoutJsonV1RenderingControllerTest {
+    @InjectMocks LayoutJsonV43RenderingController controller;
+    @Mock private HttpServletRequest req;
+
+    @Mock private HttpServletResponse res;
+
+    @Mock private IPortalRenderingPipeline portalRenderingPipeline;
```
Comment (bjagg) [inline uPortal-api/uPortal-api-rest/src/test/java/org/apereo/portal/rest/layout/LayoutJsonV1RenderingControllerTest.java]: Spacing?

**Pull request merged**